second_problem.rb

```diff
@@ -0,0 +1,32 @@
+class SecondProblem
+
+  attr_reader :never_ever_used_value
+
+  def initialize(numbers_array)
+    @numbers = numbers_array
+  end
+
+  def getBiggestTwoNumbers
+    minValue = -1234567890
+    if @numbers.empty?
+    then
+      return MinMax.new(minValue, minValue)
+    end
+
+    firstLarge = minValue
+    secondLarge = minValue
+    @numbers.each do |number|
+      if number > secondLarge then
+        secondLarge = number
+        if secondLarge > firstLarge
+          temp = firstLarge
+          firstLarge = secondLarge
+          secondLarge = temp
+        end
+      end
+    end
+
+    return MinMax.new(secondLarge, firstLarge)
```

Comment:
Interesting solution, but what happens for the following array [4, 4, 3, 2]? Can you argue that [4. 3] is not the correct solution?

Reply:
My solution would print 4 ,4 
![image](https://user-images.githubusercontent.com/16307324/39524856-9b1fe6f2-4e1a-11e8-84d8-11fc85b3ba4d.png)

Reply:
added a unit test to satisfy this situation